internal/pkg/service/stream/storage/level/local/diskwriter/network/transport/client_conn.go

```diff
@@ -87,6 +98,12 @@ func (c *ClientConnection) Close() (err error) {
 	c.lock.Lock()
 	defer c.lock.Unlock()
 
+	if c.isClosed() {
+		return nil
+	}
+
+	close(c.closed)
```

Comment:
Stream can be closed from both sides, mainly in tests, when the client and server nodes terminate in the same time.

We always call the termination of the stream/connection, even if it could have been terminated in the meantime by the other side.

It makes sense?